### ISSUE TITLE
Wait about a few seconds while disconnecting

### ISSUE THREAD

**Issue by Jeff-67** [NTU-SP/SP-Hw1-release#39]
Is it acceptable to wait about 1-2 seconds for the server to receive ret=0 when the client disconnected?

**Comment by imingasco** [comment]
What's the reason?

**Comment by Jeff-67** [comment]
I found it. 
If I close the terminal window to let it disconnect then I need to wait. But if I use kill-process, there is no need to wait. 
Will this closing window problem occur in the test cases? Or it will use "kill" in the test case?

**Comment by imingasco** [comment]
No. In the test case it works like closing the socket, but there's no need to wait too.
